fcrepo-kernel/src/main/java/org/fcrepo/kernel/observer/FedoraEvent.java

```diff
@@ -59,6 +65,7 @@
      */
     public FedoraEvent(final Event e) {
         checkArgument(e != null, "null cannot support a FedoraEvent!");
+        eventID = pidMinter.mintPid();
```

Comment:
Or we could do something like HttpClient useSystemProperties -- in this case, that would probably mean one property to specify the class to use, and then an expectation that the PidMinter instances would implement a useSystemProperties method, with a customizable suffix so you could use the same class for events and nodes with different parameters.